src/pages/thank-you.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import { Link } from 'gatsby'
+import { Flex, Box, P, H1, css } from 'bricks'
+
+const ThankYou = () => {
+  
+  const CenteredDiv = styled(Flex)(
+    css({
+      width: "100vw",
+      height: "100vh",
```

Comment:
can't you assign `width={1} height={1}` props

Reply:
`width={1} height={1}` is not equivalent to `width="100vw" height="100vh"`